ui-trackoccupancydiagram/rollup.config.js

```diff
@@ -0,0 +1,3 @@
+import generateBaseRollupConfig from '../rollup-base.config.js';
+
+export default generateBaseRollupConfig('osrdcore', ['react']);
```

Comment:
```suggestion
export default generateBaseRollupConfig('osrdTrackOccupancyDiagram', ['react']);
```

The project name in rollup.config.js should be unique to avoid global conflicts, especially for UMD/IIFE formats. It's not critical in our case since we use ESM format, but it's cleaner this way. See the doc: [https://rollupjs.org/guide/en/#outputname](vscode-file://vscode-app/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)

Reply:
yep, we missed that one, thanks

it's done